Apps/Playground/CMakeLists.txt

```diff
@@ -0,0 +1,171 @@
+set(SCRIPTS
+    "Scripts/experience.js"
+    "Scripts/playground_runner.js")
+set(BABYLONJS
+    "../Babylonjs/babylon.glTF2FileLoader.js"
+    "../Babylonjs/babylon.max.js"
+    "../Babylonjs/babylonjs.materials.js")
```

Comment:
I don't like the Babylonjs folder name. Can we just call it Scripts?

Reply:
Not sure. IMHO 'Scripts' is a fine name for subfolders in ValidationTests and Playground where it contains...well..scripts. But for Babylonjs engine, I'd like a more expressive name. Like 'BabylonEngine', 'Engine' or 'Babylon'. It's more than just a bunch of small scripts.

Reply:
I guess I'm expecting to see scripts other than Babylon in here. Anything that is in common between the apps go here? If it's strictly Babylon, then maybe BabylonScripts as the name?

Reply:
If it's common between apps and not part of Babylon, then it goes to another directory. Do you have a use case in mind?

Reply:
Do we consider 3rd party scripts (like ammo or Draco) a Babylon.js script?

Reply:
I'd consider ammo/draco to be part of the engine because they are a dependency of babylon

Reply:
Okay, are you okay BabylonScripts? I'm okay with Babylon also. Note that at some point we probably should create this script by pulling down from NPM and building a custom version using tree shaking.